doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -9,258 +9,151 @@ well as configure hypervisor capabilities and provision VMs.
-resource allocated to them. For post-launched VMs, the number of vCPUs define
-the upper limit the Service VM can allocate to them and settings of virtual
-devices still apply. Other resources are under the control of the Service VM and
-can be dynamically allocated to post-launched VMs.
+The scenario configuration defines User VMs as follows:
 
-The scenario configuration is used by ACRN configuration tool to reserve
-sufficient memory for the hypervisor to manage the VMs at build time, as well as
-by ACRN hypervisor to initialize its capabilities and set up the VMs at runtime.
+* For pre-launched User VMs, the scenario configuration defines all attributes
+  and resources (these VMs have static configurations by nature). The VM
+  attributes and resources are exactly the amount
+  of resources allocated to them.
 
-Launch Configuration
-====================
+* For post-launched User VMs, the scenario configuration defines only static
+  attributes and resources. Other resources are under the control of the
+  Service VM and can be dynamically allocated to these VMs via launch
+  scripts.
 
-The launch configuration defines the attributes and resources of a
-post-launched VM. The main contents are similar to the VM attributes and
-resources in scenario configuration. The launch configuration is used to generate shell scripts that
-invoke ``acrn-dm`` to create post-launched VMs. Unlike board and scenario
-configurations used at build time or by ACRN hypervisor, launch
-configuration are used dynamically in the Service VM.
+Launch Configuration File for Launch Scripts
+============================================
+
+The launch configuration file applies only to scenarios that have
+post-launched User VMs. The file defines certain attributes and
+resources of the post-launched VMs specified in the scenario configuration
+file. We call these settings "dynamic" because they are used at runtime.
+
+You need a launch configuration file to generate a launch script (shell script) for each post-launched User VM. The launch script invokes the
+Service VM's :ref:`Device Model <hld-devicemodel>` ``acrn-dm`` to create
+the VM. Unlike board and scenario configurations used at build time or by
+ACRN hypervisor, launch configurations are used dynamically in the Service VM.
 
 .. _acrn_config_workflow:
 
 Using ACRN Configuration Toolset
 ********************************
 
-ACRN configuration toolset is provided to create and edit configuration
-data. The toolset can be found in ``misc/config_tools``.
+The ACRN configuration toolset enables you to create
+and edit configuration data. The toolset consists of the following:
 
-Here is the workflow to customize ACRN configurations using the configuration
-toolset.
+* :ref:`Board inspector tool <board_inspector_tool>`
+* :ref:`ACRN configurator tool <acrn_configurator_tool>`
 
-#. Get the board info.
+As introduced in :ref:`overview_dev`, configuration takes place at Steps 3 and 4 in the overall development process:
```

Comment:
Since it's possible that these step numbers can change, it's better not to reference them by number and instead use a ``:ref:`name` `` link to the sections in that doc.